### DISCUSSION TITLE
Unable to connect to your device with those details. It could be an intermittent issue, or they may be incorrect.

### DISCUSSION THREAD

**Discussion by manjotsc** [make-all/tuya-local#575] [Q&A]
Hi,

I am trying to add Smart Bulb, but I am keep getting this error "Unable to connect to your device with those details. It could be an intermittent issue, or they may be incorrect."  in logs I am getting error about specifing IP address, but if I provide IP address, UI shows the same error about "Unable to connect..." and no logs.

Logs: 

<class 'Exception'>: Unable to find device on network (specify IP address) while initialising device eb7fdb5e097952ed671111

**Comment by make-all** [comment]
See topic #522
